### PR TITLE
Added user sign-up and sign-in with feature and model tests

### PR DESCRIPTION
**Pull request by drsoph**
- Edited the `user` table/model to include `first_name`, `last_name`, `bio`, `profile_photo`, `role (default: member)`

- Updated gem for factory bot and rails-helper to work with FactoryBot

- Wrote Feature and Model tests for `user`

@AL6981

### CODE REVIEW

**Comment by AL6981** [comment]
Whoever has this branch on their computer:
In your branch in the terminal run `git pull origin master` and then use your Atom to resolve the conflicts. (ask me if you need help)

**Comment by drsoph** [comment]
Updated merge conflicts
@AL6981

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+    end
+
+    it "is valid without a profile photo" do
+      expect(User.create(first_name: "Jane", last_name: "Doe", email: "jane@doe.com", bio: "Jane Doe's Bio", password: "abcd1234")).to be_valid
+    end
+
+    it "is valid with valid attributes" do
+      expect(User.create(first_name: "Jane", last_name: "Doe", email: "jane@doe.com", bio: "Jane Doe's Bio", profile_photo: "http://janedoe.com/jane.jpg", password: "abcd1234")).to be_valid
+    end
+  end
```
Comment (nwalberts) [inline spec/models/user_spec.rb]: Please DRY this code up by creating your users at the top of the describe block

YOu should also be using the let keywork as well

**Comment by drsoph** [comment]
Moved creation of user objects to top of describe block? (not sure if this was what you meant..)

**Pull request merged**